diff -ur fvwm-2.5.21~/bin/fvwm-menu-directory.in fvwm-2.5.21/bin/fvwm-menu-directory.in
--- fvwm-2.5.21~/bin/fvwm-menu-directory.in	2006-11-21 03:10:33.000000000 -0500
+++ fvwm-2.5.21/bin/fvwm-menu-directory.in	2007-02-28 21:22:40.000000000 -0500
@@ -24,7 +24,7 @@
 my $dir   = $home;
 my $xterm = "xterm -e";              # the X terminal program to invoke
 my $execT = $ENV{'SHELL'} || '/bin/sh';
-my $execF = $ENV{'EDITOR'} || "vi";  # the command to execute on plain files
+my $execF = "mimeopen -n";  # the command to execute on plain files
 my $execA = undef;
 my $commT = undef;
 my $commF = undef;
