diff -ur fvwm-2.5.21~/bin/fvwm-menu-headlines.in fvwm-2.5.21/bin/fvwm-menu-headlines.in
--- fvwm-2.5.21~/bin/fvwm-menu-headlines.in	2006-03-25 11:15:39.000000000 -0500
+++ fvwm-2.5.21/bin/fvwm-menu-headlines.in	2007-02-28 21:16:54.000000000 -0500
@@ -192,7 +192,7 @@
 my $name  = undef;
 my $title = undef;
 my $itemF = '%h\t%[(%Y-%m-%d %H:%M)]';
-my $execF = q(netscape -remote 'openURL(%u, new-window)' || netscape '%u');
+my $execF = q(xdg-open '%u');
 my $commF = undef;
 my $iconT = '';
 my $iconI = '';
