--- fvwm-2.5.21/fvwm/ConfigFvwmSetup.xdg	2007-01-23 11:13:32.000000000 +0100
+++ fvwm-2.5.21/fvwm/ConfigFvwmSetup	2007-01-25 12:14:26.000000000 +0100
@@ -90,7 +90,6 @@
 DestroyMenu MenuFvwmRoot
 AddToMenu MenuFvwmRoot  "$[gt.Root Menu]"             Title
 +                       "&1. XTerm"             Exec exec xterm
-+                       "&2. Rxvt"              Exec exec rxvt
 +                       ""              Nop
 +                       "&R. $[gt.Remote Logins]"     Popup MenuFvwmLogins
 +                       ""              Nop
@@ -106,19 +105,7 @@
 +                       "&X. $[gt.Exit Fvwm]" Popup MenuFvwmQuitVerify
 
 DestroyMenu MenuFvwmUtilities
-AddToMenu MenuFvwmUtilities     "$[gt.Utilities]" Title
-+                       "&T. Top"       Exec exec xterm -T Top -n Top -e top
-+                       "&C. Calculator" Exec exec xcalc
-+                       "&M. Xman"      Exec exec xman
-+                       "&G. Xmag"      Exec exec xmag
-+                       "&R. Editres"   Exec exec editres
-+                       ""              Nop
-+                       "&E. XEmacs"    Exec exec xemacs
-+                       "&A. Xmh Mail"  FuncFvwmMailXmh xmh "-font fixed"
-+                       ""              Nop
-+                       "&L. XLock"     Exec exec xlock -mode random
-+                       ""              Nop
-+                       "&D. $[gt.Reset X defaults]" Exec xrdb -load $HOME/.Xdefaults
+PipeRead "fvwm-xdg-menu -f -m MenuFvwmUtilities /etc/xdg/menus/applications.menu"
 
 DestroyMenu MenuFvwmConfig
 AddToMenu MenuFvwmConfig "$[gt.Fvwm Config Ops]" Title
