Index: fvwm-2.6.5/bin/fvwm-menu-headlines.in
===================================================================
--- fvwm-2.6.5.orig/bin/fvwm-menu-headlines.in
+++ fvwm-2.6.5/bin/fvwm-menu-headlines.in
@@ -208,7 +208,7 @@ my $site  = undef;
 my $name  = undef;
 my $title = undef;
 my $itemf = '%h\t%[(%Y-%m-%d %H:%M)]';
-my $execf = q(firefox '%u');
+my $execf = q(www-browser '%u');
 my $commf = undef;
 my $icont = '';
 my $iconi = '';
